tests/unit/lms/views/lti/oidc_test.py

```diff
@@ -34,7 +35,7 @@ def test_it(self, lti_registration, pyramid_request):
                 "prompt": "none",
                 "login_hint": pyramid_request.parsed_params["login_hint"],
                 "lti_message_hint": pyramid_request.parsed_params["lti_message_hint"],
-                "client_id": pyramid_request.parsed_params["client_id"],
+                "client_id": "REGISTRATION_CLIENT_ID",
```

Comment:
```suggestion
                "client_id": lti_registration.get.return_value.client_id,
```
?

If this works, I don't think you need the line above